frontend/src/components/SearchPage.js

```diff
@@ -6,15 +6,58 @@ import { Button, Modal } from 'react-bootstrap';
 import MatchingService from "../services/matchingService.js"
 
 const SearchPage = () => {
+    const [showReturnToInterview, setShowReturnToInterview] = useState(false)
     const [showCancelFindMatch, setShowCancelFindMatch] = useState(false)
+    const [timerKey, setTimerKey] = useState(0)
 
     const location = useLocation()
     const navigate = useNavigate()
 
     const difficulty = location.state.difficulty
     const username = location.state.username
 
+    const handleCloseReturnToInterview = () => setShowCancelFindMatch(false)
+    const handleShowReturnToInterview = () => {
+        setShowReturnToInterview(true)
+    }
+    const handleReturnToInterview = () => {
+        MatchingService.getInterviewByUsername(username)
+            .then(res => {
+                console.log("returning to existing interview")
+                navigate("/interview", { state: { interviewId: res.data.interviewId, username: username } })
+            })
+    }
+    const handleDeleteInterview = () => {
+        MatchingService.getInterviewByUsername(username)
+            .then(res => {
+                const interviewId = res.data.interviewId
```

Comment:
The object returned in useEffect uses _id instead of interviewId.
interviewId returns undefined instead.

![useeffectres](https://user-images.githubusercontent.com/80625519/199454766-dc1bf0cf-1ec1-4a99-9629-a6578da3168c.PNG)


If not when reconnecting to the interview page, does not load the interview so question will be blank, cannot rejoin socket rooms